backend/python/app/services/entity_service.py

```diff
@@ -0,0 +1,46 @@
+from ..models.entity import Entity
+from ..models import db
+
+
```

Comment:
General comment on error handling, I think we should throw an exception when the DB result is None, then the controller layer can catch it and send the appropriate HTTP response. We should also add a logger (can be in separate PR) - I can help look into this